all_webhooks.go

```diff
@@ -6,7 +6,6 @@ package actionlint
 // script at ./scripts/generate-webhook-events based on
 // https://github.com/github/docs/blob/main/content/actions/reference/events-that-trigger-workflows.md .
 var AllWebhookTypes = map[string][]string{
-	"branch_protection_rule":      {"created", "edited", "deleted"},
```

Comment:
This removal seems correct.

https://github.com/github/docs/commit/179a6d334e92b9ade8626ef42a546dae66b49951